servers/cromwell/README.md

```diff
@@ -4,10 +4,10 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
 
```

Comment:
Will the information in this readme be sufficient for anyone to use the stock deployment to deploy their own job manager for use (not for development)?

Reply:
I think it is tricky for users to get a clear understanding of how to use/deploy JMUI based on this readme (combine with the UI readme), so it is still important to me to add a `deploy` folder to provide template configs and deployment facing readme. I could add a minimal doc along with this PR, but it makes sense to separate dsub and Cromwell deployment instructions into several PRs.

Reply:
Excellent, I'd love if that folder could exist with some basic information about what you've implemented here and placeholders for what we need to make. That'd be very helpful!